include/onnx-mlir/Runtime/OMTensor.h

```diff
@@ -206,7 +210,7 @@ static inline int getDataTypeSize(OM_DATA_TYPE dataType) {
  * @param tensor pointer to the OMTensor
  * @return the total size of the data buffer in bytes.
  */
-int64_t omTensorGetDataBufferSize(OMTensor *tensor);
+int64_t omTensorGetDataSize(OMTensor *tensor);
```

Comment:
@tjingrant will confirm, I had the recollection that there were 2 sizes: one padded and the other one not padded. The "buffer" meant the size of the actual data plus what was needed for the padding. If that recollection is correct, regardless of the naming scheme, the distinction between the two sizes must be quite clear. 

So for the big picture: are you suggesting to use "shape" for the actual data size, and "data size" for the actual+padding? That make sense, I would suggest a explicative comment somewhere that summarize this naming scheme.

Reply:
Shape, which IMHO is a much better word, used to be called dataSizes. It's the tensor dimensions from which you can compute the tensor size in terms of the number of elements. And we had to use dataBufferSizes to denote the data size in terms of buffer size in bytes. I think the (shape, dataSize) pair is much better than (dataSizes, dataBufferSizes) pair.

Reply:
BufferSize sounds fine.